saleor/core/jwt.py

```diff
@@ -115,8 +126,30 @@ def get_user_from_payload(payload: Dict[str, Any]) -> Optional[User]:
     return user
 
 
+def is_token_belong_to_saleor(token: str) -> bool:
```

Comment:
Maybe `is_saleor_token`? Sorry for nitpicking, but the current name sounds somewhat incorrect.

Reply:
You are right